crates/mun_runtime/tests/threading.rs

```diff
@@ -0,0 +1,5 @@
+use mun_runtime::Runtime;
+
+// Ensures the [`Runtime`] is Send
+trait IsSend: Send {}
```

Comment:
Can't you directly implement `Send` for the `Runtime` class?

Also, should this be in the _tests_ folder?